### PR TITLE
fish: add page

### PR DESCRIPTION
**Pull request by deoxys314**
Added commands for the Fish shell. The information included was based
off of the information available for other shells, notably bash.

----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

File: pages/common/fish.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# fish
+
+> A command-line shell designed to be user firendly.
```
Comment (agnivade) [inline pages/common/fish.md]: Maybe mention the acronym in the first line. And then give the details. How about 
```
> Friendly Interactive SHell.
> A user friendly command line interpreter.
```

File: pages/common/fish.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+- Check a file for syntax:
+
+`fish --no-execute {{file.fish}}`
+
+- Display version information and exit:
+
+`fish --version`
+
+- Enable debug information (ranges from 0 to 128):
```
Comment (agnivade) [inline pages/common/fish.md]: This seems like an advanced option. Is this necessary to be included ? Keep in mind that tldr pages should only contain commands that are used very commonly and aimed at beginners.

Comment (deoxys314) [reply pages/common/fish.md]: It's quite useful to _me_, but is a bit higher-level than most examples, yes. removed.

File: pages/common/fish.md (hunk 1 above)
Comment (deoxys314) [reply pages/common/fish.md]: Agreed, changed.

File: pages/common/fish.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+- Execute a command:
+
+`fish -c "{{command}}"`
+
+- Run commands from a file:
+
+`fish {{file.fish}}`
+
+- Check a file for syntax:
```
Comment (sbrl) [inline pages/common/fish.md]: Did you mean ...`for syntax errors`? It might read better that way.

**Comment by sbrl** [comment]
Looks ok to em! Thanks, @deoxys314 :D

**Pull request merged**